### ISSUE TITLE
Make table-expand button WCAG2AA compliant.

### ISSUE THREAD

**Issue by DaveParillo** [danirus/sphinx-nefertiti#106]
Pa11y whines about Nefertiti’s table-expand button.
Specifically:

> Error: Anchor element found with a valid href attribute, but no link
>  content has been supplied.
>  WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.A.NoContent

The offending markup is roughly:

```html
  <a class="btn btn-outline-primary btn-sm nftt-expand" href="#">
    <i class="bi bi-arrows-angle-expand"></i>
  </a>
```
  
The icon element has no accessible name.

The fixed anchor could look like:

```html
  <a
    class="btn btn-outline-primary btn-sm nftt-expand"
    href="#"
    aria-label="Expand table"
    title="Expand table"
  >
    <i class="bi bi-arrows-angle-expand" aria-hidden="true"></i>
  </a>
```

**Comment by danirus** [comment]
Thank you @DaveParillo!

If you feel like sending a PR, please, don't hesitate. Otherwise I will do it.

**Comment by DaveParillo** [comment]
I'm a bit hesitant only because I'm largely ignorant of your theme, but I'll take a crack at it.

**Comment by DaveParillo** [comment]
Minor comment: your docs ask contributors to add the accessibility label to issues. I do not appear to have permission to add any labels to issues.

Also, I did test this with python 3.14 and sphinx 9.1, not sure if you want to add that to your tox env.

**Comment by danirus** [comment]
Thanks @DaveParillo, I just see your PR and it looks good.
I use python3.14 nowadays too. Sorry about the label, I added it myself.
If you have updated the `tox.ini` it would be great to have it in the PR too.

**Comment by danirus** [comment]
@DaveParillo, thank you so much for your contribution.
I have created a new release, v0.9.9, with your changes.

**Comment by DaveParillo** [comment]
I was not completely sure f968ce7 was completely finished - I did some more work you may or not be ok with.

Mostly I was concerned about the tox matrix combinations and tried being more precise about the Sphinx python compatibility. Use it or not as you see fit.

**Comment by danirus** [comment]
Sorry, I thought it was ready. Precision is good, so please send the changes in another PR when they are ready. If the changes are about tox, they won't be needed in the package distribution created for each release, but they are worth having here in the repository, as part of the CI process.
